Aptoide/appcoins-wallet-android
# [bugfix/MOB-1045] Missing Alpha on Future Perk Icon

**What does this PR do?**

  Simply makes the icon of the future perk faded as it was initially specified.

**Database changed?**

   No

**How should this be manually tested?**

  Simply insert a FutureItem promotion (I simply hardcoded this in the PromotionsInteractor). Then, go to promotions page and check if the future promotion looks the same as in Zeplin.

**What are the relevant tickets?**

  [MOB-1045](https://aptoide.atlassian.net/browse/MOB-1045)


**Code Review Checklist**
